server/app/views/admin/TranslationFormView.java

```diff
@@ -96,7 +103,13 @@ protected final FormTag renderTranslationForm(
                             "Save %s updates",
                             locale.getDisplayLanguage(LocalizedStrings.DEFAULT_LOCALE)))
                     .withId("update-localizations-button")
-                    .withClasses(ButtonStyles.SOLID_BLUE));
+                    .withClasses(ButtonStyles.SOLID_BLUE),
+                div(a().withHref(redirectUrl)
+                        .with(
+                            button("Back")
+                                .withId("back-to-edit")
+                                .withClasses(ButtonStyles.OUTLINED_TRANSPARENT)))
+                    .withClasses("mt-4"));
```

Comment:
(blocking) Place the buttons on the same line with the back button first. The html should be something like

```html
<div class="flex flex-row">
   <button>Back</button>
   <button>Save</button>
</div>
```

Reply:
Done!